# Text Compare
# Left: /l/left.txt (UTF-8, lf)
# Right: /r/right.txt (UTF-8, lf)
# Mode: display: context (3 lines); ignore unimportant: no
# Rules: format: Everything Else; algorithm: standard (auto); skew: 300; closeness: 500‰; never align: no; unimportant: whitespace, case; line endings: ignored; orphan lines: always important
#
*** /l/left.txt	2026-10-19T13:12:00Z
--- /r/right.txt	2026-10-19T14:00:00Z
***************
*** 1,7 ****
  int main() {
!     return 0;
  }
! // Comment Text
! value = 1
  same line
! other  spaced
--- 1,8 ----
  int main() {
!         return 0;
  }
! // comment text
! value = 2
  same line
! other spaced
! new line
